src/main/webapp/web.xml

```diff
@@ -11,4 +12,92 @@
         <servlet-name>index</servlet-name>
         <url-pattern>/index</url-pattern>
     </servlet-mapping>
+    <servlet-mapping>
```

Comment:
better structure a little bit in a different way
files in views:
car/all.jsp
car/add.jsp
...
driver/all.jsp
driver/add.jsp
and so on

similar with URLs 
/cars/all
/cars/create
..
/manufacturer/all
..